pom.xml

```diff
@@ -468,7 +386,7 @@
                         <includes>
                             <include>**/*Test.java</include>
                         </includes>
-                        <argLine>-XX:-OmitStackTraceInFastThrow -Xms1024m -Xmx2048m @{argLine}</argLine>
+                        <argLine>-XX:-OmitStackTraceInFastThrow -Xms1024m -Xmx2048m --add-opens java.base/java.lang=ALL-UNNAMED @{argLine}</argLine>
```

Comment:
This bit was a surprise to me. Jacoco must have been doing this on our behalf. Without this, unit tests and serialization tests started to fail in Java 17 and higher when calling clone().